department-of-veterans-affairs/next-build
# Add events query/formatter/layout

## Description
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15908

Adds Events to next-build.

Changes:
- Adds events query and formatter
- formatter and query tests and mock data
- Event layout
- Event utl functions for date calculations
- Unit tests
- Google maps component
- Adjusts social links component
- Adds to storybook

## Testing done
Local testing, unit tests

## Screenshots
<img width="1093" alt="image" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/0ba7dab8-0522-4195-a65a-4bdcdeaad7e4">


## QA steps
Verify that event nodes (ie http://localhost:3000/central-iowa-health-care/events/52265/) render as expected
Verify that events data is properly brought in


## Is this PR blocked by another PR?
- Add the `DO NOT MERGE` label
- Add links to additional PRs here:
